new/LIT6833Salam.xml

```diff
@@ -0,0 +1,66 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6833Salam" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Salām lakkəmu Ṣādəqān wa-Samāʿət</title>
```

Comment:
samāʿt
I also would spell all words after the first small since they are not really proper names - but I'm not sure

Reply:
I was wondering, if I delete the ə in samāʿ(ə)t, I need also delete the ə in ṣād(ə)qān, not? If it is no disturbance to anyone, I can be more archaic and less traditional in this.
No, they are not proper names. So, if this is the rule, I can write all translitterations in uniform small letters.

Reply:
I think that both shwas are not etymologically equivalent - samāʿt is the plural pattern qatalt, and the a is only long due to the laryngeal, while in ṣādəqān the shwa is part of the pattern, and I'm not aware of a vowel reduction. But I'm open to other interpretations!

(There is this list based on decisions taken in TraCES, but it's not really updated regularly: https://betamasaheft.eu/Guidelines/?id=dubious-spelling)

Reply:
So, I am convinced to spell ṣādəqān, but samāʿt.

Reply:
It might be useful to have a longer list in the guidelines, for instance for lakkəmu vs. lakəmu etc.

Reply:
That's true. The basic assumption is that the TraCES system is used, but there are some differences, and updates might always be necessary. Feel free to edit the guidelines! Alessandro could then also be assigned in a guidelines PR